src/com/lftechnology/java/training/srijan/basics/football/ManchesterUnitedStats.java

```diff
@@ -0,0 +1,229 @@
+ * </p>
+ * <p>
+ * Suppose a win gains you 3 points, a draw 1 point, and a loss no points. Have
+ * your programme work out how many points in total Manchester United have
+ * acquired.
+ * </p>
+ * <p>
+ * When you have finished this exercise, the output in your textbox, listbox or
+ * console should be as follows:
+ * </p>
+ * <p>
+ * No. of wins = 3
+ * </p>
+ * <p>
+ * No. of draws = 1
+ * </p>
+ * <p>
+ * No. of defeats = 1
+ * </p>
+ * <p>
+ * Goal Scored = 10
+ * </p>
+ * <p>
+ * Goal Conceded = 6
+ * </p>
+ * <p>
+ * No. of points = 10
+ * </p>
+ * 
+ * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+ *
+ */
+public class ManchesterUnitedStats {
+	private static final Logger LOGGER = Logger.getLogger(ManchesterUnitedStats.class.getName());
+	private String[] fixtureWithScore;
+	private String clubName;
+
+	public ManchesterUnitedStats(int numberOfGameWeeks, String club) {
+		this.fixtureWithScore = new String[numberOfGameWeeks * 4];
+		this.clubName = club;
+		for (int i = 0; i < fixtureWithScore.length; i++) {
+			fixtureWithScore[i] = "";
+		}
+	}
+
+	/**
+	 * <p>
+	 * Displays Manchester United's fixtures
+	 * </p>
+	 * 
+	 * @param numberOfMatches
+	 *            {@link Integer}
+	 * @param gamesFixtures
+	 *            {@link String}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void displayGameweekFixtures(int numberOfMatches, String[] gamesFixtures) {
+		int count = 0;
+		LOGGER.info(clubName + " Fixtures:\n ");
+		for (int i = 0; i < gamesFixtures.length; i += 2) {
+			if (count < 2) {
+				LOGGER.info(gamesFixtures[i] + " vs " + gamesFixtures[i + 1] + "\n");
+				count++;
+			}
+			count = 0;
+		}
+	}
+
+	/**
+	 * <p>
+	 * Inputs Match Scores
+	 * </p>
+	 * 
+	 * @param gamesFixtures
+	 *            {@link String}
+	 * @param numberOfMatches
+	 *            {@link String}
+	 * @author Srijan Bajracharya <srijanbajracharya@lftechnology.com>
+	 */
+	public void inputGameweekScores(int numberOfMatches, String[] gamesFixtures) {
+		Scanner inputScore = new Scanner(System.in);
+		String scores = "";
+		int arrayCounter = 0;
+		int team = 0;
+
+		try {
+			for (int i = 1; i <= numberOfMatches; i++) {
+
+				LOGGER.info("GameWeek" + i);
+				LOGGER.info(gamesFixtures[team] + " vs " + gamesFixtures[team + 1]);
+				this.fixtureWithScore[arrayCounter] = gamesFixtures[team];
+				LOGGER.info(gamesFixtures[team] + " score: ");
+
+				if (inputScore.hasNext()) {
+					scores = inputScore.nextLine();
+					this.fixtureWithScore[++arrayCounter] = scores;
+					LOGGER.info(gamesFixtures[team + 1] + " score: ");
+					this.fixtureWithScore[++arrayCounter] = gamesFixtures[team + 1];
+					scores = inputScore.nextLine();
+					this.fixtureWithScore[++arrayCounter] = scores;
+				}
+				team = team + 2;
+				arrayCounter++;
+			}
+
+		} catch (NoSuchElementException ex) {
+			LOGGER.info("Error Message:" + ex.getMessage());
+		} catch (ArrayIndexOutOfBoundsException exception) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)